README.markdown

```diff
@@ -10,9 +10,17 @@ Test implementation of RESTful API with Dropwizard.
 
 ## Build
 
-Define database connection in `configuration.yaml`.
+Define database connection in `configuration.yaml`:
 
-Compile application into single Java archive:
+To connect to the database, you'll need to download `ojdbc6_g.jar` from Oracle's website.
+
+Link:  http://www.oracle.com/technetwork/database/enterprise-edition/jdbc-112010-090769.html
+
+You must accept Oracle's terms and conditions and the download can be found a few lines below the terms of agreement. After downloading the jar, in your main directory, create a `bin` file and move the jar into it. Next you'll need to modify `configuration.yaml`.  Under the `database:` section, update the `user:`, `password:`, and `url:` sections with your credentials.
```

Comment:
put the instructions for modifying the configuration file in a separate paragraph.